modules/mbedtls/zephyr_init.c

```diff
@@ -104,7 +104,7 @@ static int _mbedtls_init(void)
 }
 
 #if defined(CONFIG_MBEDTLS_INIT)
-SYS_INIT(_mbedtls_init, POST_KERNEL, 0);
```

Comment:
There are dozens of other SYS_INIT call sites using priority level 0. Do you want to fix them all? Should we make this an error so that we can catch it in the future?

Reply:
Fixing them all needs fully test, but now I only can find the issue between mbedtls and malloc on our device, so for the safety consideration, I will only change this one.